src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,108 @@
+import Pagination from './Pagination';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+test('One page - both next and previous button disabled', () => {
+  const params = {
+    pages: 1,
+    currentPage: 1,
+    jumpToPage: jest.fn(),
```

Comment:
cara, não é nenhum empecilho, mas você poderia ter usado um "beforeEach()" pra configurar esses params aí, em vez de configurar eles toda vez dentro de cada teste.

declara o "params" fora dos escopos de teste e, dentro de cada um deles, altera só o que for necessário. Mais ou menos assim

`describe('Page button tests, () => {
  let params;
  beforeEach(() => {
    params = {
      pages: 1,
      currentPage: 1,
      jumpToPage: jest.fn(),
      nextPage: jest.fn(),
      previousPage: jest.fn(),
    }
  }
  test('Two pages - one of next or previous button disabled', () => {
  params.pages = 2;
  params.currentPage = 1;
...
  };
}
`